src/Mod/Part/App/TopoShape.cpp

```diff
@@ -3130,7 +3130,9 @@ TopoDS_Shape TopoShape::makeOffset2D(double offset, short joinType, bool fill, b
         BRep_Builder builder;
         builder.MakeCompound(result);
         for(TopoDS_Shape &sh : shapesToReturn) {
-            builder.Add(result, sh);

```

Comment:
Can you please make a separate PR just for this. Just for technical reason to help me with the review.